### PR TITLE
chore: align for release 2.3.10

### PR DESCRIPTION
**Pull request by jacobdotcosta**
Align upstream for 2.3.10 release.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -123,30 +123,30 @@
   </pluginRepositories>
 
   <properties>
-    <amqp-10-starter.version>2.3.3</amqp-10-starter.version>
-    <dekorate.version>1.0.3</dekorate.version>
-    <infinispan-starter.version>2.3.3.Final</infinispan-starter.version>
+    <amqp-10-starter.version>2.3.6</amqp-10-starter.version>
```
Comment (aureamunoz) [inline pom.xml:126]: MMMmmm...not sure, shouldn't it be the `2.9.1` --> https://issues.redhat.com/browse/ENTMQCL-2758?

File: pom.xml [hunk 2]
```diff
@@ -123,30 +123,30 @@
...
+    <dekorate.version>2.1.3</dekorate.version>
+    <infinispan-starter.version>2.3.6.Final</infinispan-starter.version>
     <!-- https://github.com/snowdrop/narayana-spring-boot -->
     <narayana.version>5.10.6.Final</narayana.version>
-    <narayana-starter.version>2.5.0</narayana-starter.version>
+    <narayana-starter.version>2.5.3</narayana-starter.version>
     <!-- https://github.com/opentracing-contrib/java-spring-jaeger -->
     <opentracing-spring-jaeger-web-starter.version>3.2.2</opentracing-spring-jaeger-web-starter.version>
-    <resteasy.version>3.14.0.Final</resteasy.version>
+    <resteasy.version>3.6.4.Final</resteasy.version>
```
Comment (aureamunoz) [inline pom.xml]: In [this jira](https://issues.redhat.com/browse/EAPSUP-502) it's said that resteasy-jaxrs shoud be 3.15.1, why a different version of the one indicated [here](https://issues.redhat.com/browse/RESTEASY-2873)?

Comment (jacobdotcosta) [reply pom.xml]: You're right, I used the starter version instead of the actual resteasy version.

Comment (jacobdotcosta) [reply pom.xml]: Fixed.

File: pom.xml (hunk 1 above)
Comment (jacobdotcosta) [reply pom.xml:126]: I don't think so because that's the product version and here we're only applying the starter.

**Review by aureamunoz — APPROVED** [review]
LGTM

**Pull request merged**